### DISCUSSION TITLE
Does bentoml support multiple aws credentials?

### DISCUSSION THREAD

**Discussion by HenryDashwood** [bentoml/BentoML#1132] [General] [answered]
I have a few AWS accounts so my `.aws/credentials` file looks something like this. For working directly with AWS I can specify which account to use with the `--profile` flag. What's the best way of using bentoml with a non default AWS account? 
```
[default]
aws_access_key_id = *****************
aws_secret_access_key = *****************
[another_profile]
aws_access_key_id = *****************
aws_secret_access_key = *****************
[yet_another_profile]
aws_access_key_id = *****************
aws_secret_access_key = *****************
```

**Comment by yubozhao** [comment, marked as answer]
Hey @HenryDashwood, good to see you again!

Right now, It doesn't support choose which AWS account.  We probably should provide this option, do you mind open a feature request for this?

The current workaround is to set your AWS profile as an environment variable before you run the bentoml deployment commands.
i.e. ```$ export AWS_PROFILE=another_profile & bentoml lambda/sagemaker deploy ....```

edit for typo

**Comment by HenryDashwood** [comment]
Perfect, will do!
